app/controllers/api/v1/cars.rb

```diff
@@ -0,0 +1,25 @@
+class Api::V1::CarsController < ApplicationController
+  before_action :set_cars, only: %i[show destroy]
```

Comment:
Please only configure actions for routes you are working on
```suggestion
  before_action :set_cars, only: %i[show]
```

Reply:
Hello  @Shisui6 
Thanks for your kind review and helpful feedback